js/ui/form/components/field_item.js

```diff
@@ -112,8 +112,13 @@ export function renderFieldItem({
         const isToggleControls = toggleControls.includes(editorType);
         const labelAlignment = labelOptions.alignment;
         const isLabelAlignmentLeft = labelAlignment === 'left' || !labelAlignment;
+        const hasTemplate = Boolean(template);
+        const isLabelOnTop = labelLocation === 'top';
 
-        if(isToggleControls && labelLocation === 'top' && isLabelAlignmentLeft) {
+        if(isToggleControls
```

Comment:
I suggest to change order to (hasNoTemplate && isToggleControls && isLabelOnTop && sLabelAlignmentLeft). We don't require to check other conditions if hasNoTemplate is true.